packages/mdc-fab/README.md

```diff
@@ -121,15 +85,28 @@ You can also do this declaratively when using the [material-components-web](../m
 
 MDC FAB is fully aware of MDC Ripple styles, so no DOM or CSS changes are required.
 
-### Sass Mixins
+## Style Customization
+
+### CSS Classes
+
+CSS Class | Description
+--- | ---
+`mdc-fab` | Mandatory, for the button element
+`mdc-fab__icon` | Mandatory, for the icon element
+`mdc-fab--mini` | Optional, modifies the FAB to a smaller size
+`mdc-fab--exited` | Optional, animates the FAB out of view.<br>When this class is removed, the FAB will return to view.
 
-By default an MDC FAB will inherit its color from the theme. This mixin will override the color of the MDC FAB's container, but maintain accessibility standards for the ink and ripple. The mixin is intended for customizing an MDC FAB's color to a non-theme color.
+> **A note about `:disabled`**, No disabled styles are defined for FABs. The FAB promotes action, and should not be displayed in a disabled state. If you want to present a FAB that does *not* perform an action, you should also present an explanation to the user.
+
+### Sass Mixins
 
-#### `mdc-fab-accessible($container-color)`
+Mixin | Description
```

Comment:
Add `#### Basic Sass Mixins` above this table for consistency with Button, and add the following paragraph before the table:

```
MDC FAB uses [MDC Theme](../mdc-theme)'s `secondary` color by default. Use the following mixins to customize it.
```